docs/migration-guide/to-16.md

```diff
@@ -154,7 +161,9 @@ We've published a new version of [Jest preset](https://www.npmjs.com/package/jes
  }
 ```
 
-If you get a segmentation fault while running the preset on Node.js 18, you can use [`jest-light-runner`](https://www.npmjs.com/package/jest-light-runner):
+(`cross-env` is in maintenance-mode as it's [considered finished](https://github.com/kentcdodds/cross-env/issues/257#issue-754591323).)
```

Comment:
minor

add `> [!NOTE]`

Reply:
Let's only use those outside the migration guide, e.g. [Node.js API docs](https://stylelint.io/user-guide/node-api#output), as the migration guide has a flatter hierarchy.